pyicloud/services/findmyiphone.py

```diff
@@ -43,6 +43,7 @@ def refresh_client(self):
                         "fmly": self.with_family,
                         "shouldLocate": True,
                         "selectedDevice": "all",
+                        "deviceListVersion": 1
```

Comment:
add a coma at the en of line to fix `black` formatting

Reply:
Done